jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -255,10 +255,14 @@ public void deleteTaskComment(String serverTemplateId,
         UserTaskServicesClient client = getClient(serverTemplateId,
                                                   UserTaskServicesClient.class);
 
-        List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
-                                                                    taskId);
-
-        return comments.stream().map(c -> build(c)).collect(toList());
+        try {
+            List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
+                                                                        taskId);
+            return comments.stream().map(c -> build(c)).collect(toList());
+        } catch (KieServicesException e) {
+            // task not found
```

Comment:
Having a look at _getTaskCommentsByTaskId()_ implementation in _org.kie.server.client.impl.UserTaskServicesClientImpl_, the exception is thrown when there's an actual failure response upon executing JMS command. Are we sure that it's safe to simply swallow this exception with a comment and continue?